components/config/basedir.rs

```diff
@@ -76,7 +76,7 @@ pub fn default_config_dir() -> Option<PathBuf> {
 }
 
 #[cfg(target_os = "windows")]
-pub fn default_config_dir() -> Option<PathBuf> {
+pub fn default_config_dir() -> PathBuf {
```

Comment:
The return values for these functions weren't updated to match.

Reply:
Sorry. Overly enthusiastic first time committer :)
```
./mach build -d
./mach test-tidy
./mach test-unit
./mach run tests/html/about-mozilla.html 
./mach run https://ddg.gg
```

And it all seems fine. I guess the PR should pick up the changes now.

Reply:
@jdm It failed again in Task Cluster :(
How can I make it fail on my machine? I'm obviously missing something here.
I'm on GNU/Linux (Fedora).

Reply:
Taskcluster failures can be ignored since it always fails.

Reply:
I was really curious how I managed to break it so that `It looks like rustup is not installed.` :)

Thank you for your time!